### PR TITLE
Fb 73 ability to auth as a user from the command line

### PR DESCRIPTION
**Pull request by JakeGinnivan**

### CODE REVIEW

File: libs/code-generator/src/lib/code-generator.ts [hunk 1]
```diff
@@ -13,10 +13,10 @@ import type { Tree } from './tree/tree'
 const promptsInternal: typeof prompts =
     typeof prompts == 'function' ? prompts : require('prompts')
 
-export type TemplateType = 'dotnet-api'
+export type Template = 'dotnet-api'
```
Comment (JakeGinnivan) [inline libs/code-generator/src/lib/code-generator.ts:16]: Renamed from templateType to template

File: apps/cli/src/lib/title-text.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// https://manytools.org/hacker-tools/ascii-banner/
+export const titleText = `
+ ___         _                ___                   _    ___ _    ___
```
Comment (emlyn-rice) [inline apps/cli/src/lib/title-text.ts:3]: We seem to be missing parts of the CLI

File: libs/code-generator/src/lib/code-generator.ts (hunk 1 above)
Comment (emlyn-rice) [reply libs/code-generator/src/lib/code-generator.ts:16]: Can we remember to update the docs

**Review by emlyn-rice — APPROVED** [review]

File: apps/cli/src/lib/title-text.ts (hunk 2 above)
Comment (JakeGinnivan) [reply apps/cli/src/lib/title-text.ts:3]: image

I removed the version

File: libs/code-generator/src/lib/code-generator.ts (hunk 1 above)
Comment (JakeGinnivan) [reply libs/code-generator/src/lib/code-generator.ts:16]: That's in the separate repo?

I'll update once I add the auth stuff in

**Pull request merged**